### PR TITLE
Rename metrics `:sites` dimension with `:locations`

### PR DESCRIPTION
**Pull request by Zapiano**
I took the opportunity to update some variables that used the word "site" to use "location". That will help us migrate completely from `site`/`sites` to `location`/`locations` in the future.

Note: This is a first step before adding metadata to the metrics generated by Metrics module

### CODE REVIEW

File: src/metrics/ranks.jl [hunk 1]
```diff
@@ -199,10 +198,11 @@ YAXArray[timesteps, sites, scenarios]
...
 """
-function shade_ranks(rs::ResultSet; kwargs...)
-    selected = _get_ranks(rs, 2; kwargs...)
-    return _collate_ranks(rs, selected; kwargs...)
-end
+# There is no shade rank
+# function shade_ranks(rs::ResultSet; kwargs...)
+#     selected = _get_ranks(rs, 2; kwargs...)
+#     return _collate_ranks(rs, selected; kwargs...)
+# end
```
Comment (ConnectedSystems) [inline src/metrics/ranks.jl:205]: Delete this and the docstring above.

File: src/ExtInterface/ADRIA/Domain.jl [hunk 2]
```diff
@@ -18,13 +18,13 @@ mutable struct ADRIADomain <: Domain
     env_layer_md::EnvLayer  # Layers used
     scenario_invoke_time::String  # time latest set of scenarios were run
     const conn::YAXArray  # connectivity data
-    site_data::DataFrame  # table of site data (depth, carrying capacity, etc)
-    const site_id_col::String  # column to use as site ids, also used by the connectivity dataset (indicates order of `conn`)
-    const cluster_id_col::String  # column of unique site ids
+    site_data::DataFrame  # table of location data (depth, carrying capacity, etc)
+    const site_id_col::String  # column to use as location ids, also used by the connectivity dataset (indicates order of `conn`)
```
Comment (ConnectedSystems) [inline src/ExtInterface/ADRIA/Domain.jl:22]: `loc_id_col` ?

File: src/ExtInterface/ADRIA/Domain.jl [hunk 3]
```diff
@@ -18,13 +18,13 @@ mutable struct ADRIADomain <: Domain
...
+    const cluster_id_col::String  # column of unique cluster ids
     init_coral_cover::YAXArray  # initial coral cover dataset
     const coral_growth::CoralGrowth  # coral
-    const site_ids::Vector{String}  # Site IDs that are represented (i.e., subset of site_data[:, site_id_col], after missing sites are filtered)
-    const removed_sites::Vector{String}  # indices of sites that were removed. Used to align site_data, DHW, connectivity, etc.
+    const site_ids::Vector{String}  # Location IDs that are represented (i.e., subset of site_data[:, location_id_col], after missing locations are filtered)
```
Comment (ConnectedSystems) [inline src/ExtInterface/ADRIA/Domain.jl:26]:
```suggestion
    const site_ids::Vector{String}  # Location IDs that are represented (i.e., subset of site_data[:, loc_id_col], after missing locations are filtered)
```

Standardize to `loc_id_col`?

**Comment by Zapiano** [comment]
@ConnectedSystems  I've implemented your suggestions, and I have replaces `site`/`sites` by `loc`/`location`/`locs`/`locations` in several other places. The tests are passing and I manually loaded and run scenarios for RME as well and everything seems to be fine. There are a lot more places to change though, and my initial idea wasn't to convert every single occurrence of in this PR, but only to give an initial step. I think this PR is replacing more than 200 occurrences of `site`/`sites` already. 

That said, let me know if you think I should change anything else.

**Review by ConnectedSystems — APPROVED** [review]

**Pull request merged**